modules/oadp-installing-dpa.adoc

```diff
@@ -32,6 +32,15 @@ endif::[]
 ====
 If you do not want to specify backup or snapshot locations during the installation, you can create a default `Secret` with an empty `credentials-velero` file. If there is no default `Secret`, the installation will fail.
 ====
++
+[NOTE]
+====
+Velero creates a secret named `velero-repo-credentials` in the `velero install` namespace, which contains a default backup repository password.
```

Comment:
We should probably say "OADP" rather than `velero install` for the namespace, since velero is installed via DPA in the oadp namespace rather than using the `velero install` CLI.